frontend/views/containers/chatroom/chat-mentions/RenderMessageText.vue

```diff
@@ -74,6 +75,9 @@ export default ({
         ...Object.keys(this.ourContactProfilesById).map(u => makeMentionFromUserID(u).me).filter(v => !!v),
         makeChannelMention('[^\\s]+', true) // chat-mention as contractID has a format of `#:chatID:...`. So target them as a pattern instead of the exact strings.
       ]
+    },
+    showTrailingEllipsis () {
+      return this.isReplyingMessage && this.text.length === CHATROOM_REPLYING_MESSAGE_LIMITS_IN_CHARS
```

Comment:
Probably want `>` instead of `===` right?

Interestingly enough, this doesn't seem to prevent the `...` from being shown for long messages, and I'm not sure why.

I did notice though that the ellipses is sometimes placed on a newline when it probably should be:

![Screenshot 2025-03-20 at 10 26 13 AM](https://github.com/user-attachments/assets/b74fbb12-859f-4ecf-b8b3-e9fecd115d24)

Reply:
@taoeffect As I mentioned in [this PR description](https://github.com/okTurtles/group-income/pull/2730#issue-2933584308), truncating the replied message to the max 180 characters are done before the message is sent. This means the replied message being displayed in the message chain won't be longer than 180 characters and that's why. So I don't think this part needs a further change. Let me know otherwise though.

Reply:
Well, one thought I had is that what if some time in the future we decide to change `CHATROOM_REPLYING_MESSAGE_LIMITS_IN_CHARS` to make it shorter?

Then `...` won't be displayed after older messages that we're replying to that are longer than that. So that might be a situation where we'd still want to use `>=` just to be safe.

Also, what about the image above that shows the ellipses on a newline? Is there a way to make sure that it's not on a newline by itself like that? It's not a big deal TBH, we could leave it as-is, but as a kind of attention-to-detail polish I think it would look better if it remained on the same line as the text.

Reply:
>  I think it would look better if it remained on the same line as the text.
This was my first approach actually and then I decided to make it `display: block`. I can revert it back.

Reply:
updated so that the ellipsis is inline rather than a block.

![image](https://github.com/user-attachments/assets/c1f18f01-3e01-425a-8726-a3cabd8d9f78)